Day2/Exercise2/Solution.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"time"
+)
+
+func expectedRating(val float64, wg *sync.WaitGroup, sum *float64){
+	defer wg.Done()
+	time.Sleep(time.Second*time.Duration(rand.Intn(2)))
+	*sum=*sum+val
+}
+func main () {
+
+	var wg sync.WaitGroup
+	var students[200] float64
+	k:=0.0
+	for i:=0;i<200;i++ {
+		students[i]=float64(rand.Intn(5))
```

Comment:
Btw , rand.Intn will always produce integers so your idea of having ratings in form of floating point numbers would not be fulfilled